### ISSUE TITLE
On Import implicit affix is merged with explicit affix

### ISSUE THREAD

**Issue by SimonQR** [poeapp/poeapp#112]
### Did you clear your browser cache before trying to reproduce?
- [X ] Yes

### App Version
febf7d21b28e60edbfdf602cfb90cb30408c55a8
### Description
If i import for example a Dreamfeather using Ctrl + C in-game and paste it into the import box on the website, the implicit accuracy rating of the sword is merged with the explicit accuracy on the sword, but is tagged as an explicit rating on the mod list.

### Expected outcome
The Implicit and explicit affixes are kept separate and added as two different mods on the list, or merged and tagged with [total]

### Actual outcome
Implicit accuracy rating of 475 gets merged with explicit accuracy rating of 193 to a total of 668 accuracy rating, tagged with [explicit]

### Steps to reproduce
1. Import a Dreamfeather
2. Accept import
3. Look at the automatic mod list created by the import
4. Look away in disgust

### Browsers Affected

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE 11
- [ ] iOS Safari
- [ ] Android Chrome

**Comment by poeapp** [comment]
Can you please paste the in-game text for dreamfeather in this ticket?

**Comment by SimonQR** [comment]
Here you go
Rarity: Unique
Dreamfeather
Eternal Sword
--------
One Handed Sword
Quality: +20% (augmented)
Physical Damage: 95-169 (augmented)
Critical Strike Chance: 5.00%
Attacks per Second: 1.86 (augmented)
Weapon Range: 9
--------
Requirements:
Level: 66
Str: 104
Dex: 122 (unmet)
--------
Sockets: R G R 
--------
Item Level: 76
--------
+475 to Accuracy Rating
--------
Adds 38 to 73 Physical Damage
24% increased Attack Speed
+193 to Accuracy Rating
+186 to Evasion Rating
3% increased Movement Speed
1% increased Attack Damage per 450 Evasion Rating
--------
She dances a dance beneath the shade,
The twilight shroud her only blade,
Who now hears her feathersteps flowing?
Beneath the stars, the moondrops glowing.
--------

**Comment by SimonQR** [comment]
After further investigation it appears that they are not necessarily merged, but some implicit affixes are added as explicit mods

**Comment by poeapp** [comment]
The following item imports `2 secondary Tornado Shot Projectiles` as explicit instead of enchant due to the tshot gem.

```Rarity: Unique
Rat's Nest
Ursine Pelt
--------
Quality: +20% (augmented)
Evasion Rating: 745 (augmented)
--------
Requirements:
Level: 60 (unmet)
Dex: 114 (unmet)
--------
Sockets: R-R-R-G 
--------
Item Level: 80
--------
2 secondary Tornado Shot Projectiles
--------
15% increased Attack Speed
73% increased Global Critical Strike Chance
150% increased Evasion Rating
24% increased Rarity of Items found
10% increased Movement Speed
10% reduced Character Size
--------
A lad with no fear of the pest 
Let rats fill his helmet in jest 
When they picked his skull clean
Where his head should have been
Was naught but a vermin-filled nest!```

**Comment by poeapp** [comment]
This should be fixed as of `2018-04-04T22:02:10.867Z`.  Please comment here if you see otherwise.
